### PR TITLE
<Checkbox/> - add `selectionAreaSkin` and `selectionAreaPadding` props

### PR DESCRIPTION
**Pull request by assafganor**
#pr

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6403

File: src/Checkbox/index.d.ts [hunk 1]
```diff
@@ -14,10 +14,13 @@ export interface CheckboxProps {
   size?: CheckboxLabelSize;
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
   className?: string;
+  SelectionAreaSkin?: CheckboxSelectionAreaSkin;
+  SelectionAreaPadding?: React.CSSProperties['padding'];
```
Comment (EgoziE) [inline src/Checkbox/index.d.ts:18]: Please also add a type test for these two props (in file src/Checkbox/test/Checkbox.tsx)

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Checkbox/index.d.ts (hunk 1 above)
Comment (assafganor) [reply src/Checkbox/index.d.ts]: added and fix typos

File: src/Checkbox/Checkbox.st.css [hunk 2]
```diff
@@ -92,22 +93,29 @@
...
-.root:selectionArea(hover):hover {
+.root:selectionArea(always):selectionAreaSkin(filled):hover,
+.root:selectionArea(hover):selectionAreaSkin(filled):hover {
   background-color: value(THEME-COLOR-50);
 }
 
+.root:selectionArea(always):selectionAreaSkin(outlined):hover,
+.root:selectionArea(hover):selectionAreaSkin(outlined):hover {
+  box-shadow: 0 0 0 1px value(THEME-COLOR-20);
+}
```
[the remark this answers is not in the document]
Comment (assafganor) [reply src/Checkbox/Checkbox.st.css]: I moved the border and padding to the label so all area will be "active"

File: src/Checkbox/Checkbox.js [hunk 3]
```diff
@@ -81,49 +84,53 @@ class Checkbox extends React.PureComponent {
...
-            disabled={disabled || !hasError || !errorMessage}
-            placement="top"
-            textAlign="center"
-            content={errorMessage || ' '}
-            maxWidth={230}
-            hideDelay={150}
-            zIndex={10000}
+          <div
+            className={classes.labelInner}
+            style={{ padding: selectionAreaPadding }}
```
Comment (EgoziE) [inline src/Checkbox/Checkbox.js:94]: This extra div breaks visual tests.
Take a look at ListItemSelect component for example:
https://preview.wix-style-react.com/pr_6403/?path=/story/components-api-components--listitemselect

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by milkyfruit — APPROVED** [review]
@assafganor this PR looks great.

**Review by EgoziE — APPROVED** [review]

**Pull request merged**